core/project.clj

```diff
@@ -16,7 +16,8 @@
 
                  [com.cognitect/transit-clj "1.0.324"]
                  [com.cognitect/transit-cljs "0.8.269"]
-                 [frankiesardo/linked "1.3.0"]]
+                 [frankiesardo/linked "1.3.0"]
```

Comment:
This dep could be removed in favor of the one below since the one below works in both bb and JVM

Reply:
Happy to remove frankiesardo/linked